### ISSUE TITLE
Django 1.11 to 2.2 codemods.

### ISSUE THREAD

**Issue by charettes** [browniebroke/django-codemod#23]
### Description

Hello @browniebroke thanks for this project it looks promising! I believe it can greatly accelerate the speed of adoptions of newly released versions of Django.

I was wondering if you'd be interested in receiving contributions for migration to versions between Django 1.11 LTS and 2.2 LTS. Even if these versions are not supported anymore I think it could be of interest to a lot of developers who are still in the process of upgrading.

I currently work for a company that is the process of doing so and also has a few Django 1.9/1.10 deprecation warnings that need to be addressed on a large project. Since we'll have to automate the process anyway I was wondering if you'd be interested in receiving contributions for these versions as well.

**Comment by browniebroke** [comment]
Yes, I would definitely accept these contributions for these. We are already running 2.2, and it was not my primary use case, hence I focused on 3.0+, but I think it makes to support at least from 1.11 which has just gone out of support and I assume lots of people are still using.

**Comment by jayvdb** [comment]
Many older fixers can be found and imported from https://github.com/pakal/django-compat-patcher

c.f. https://github.com/pakal/django-compat-patcher/tree/development/src/django_compat_patcher/django_legacy and https://github.com/pakal/django-compat-patcher/tree/development/src/django_compat_patcher/fixers

I created https://github.com/pakal/django-compat-patcher/issues/12 about potentially sharing metadata between the two projects.
